### PR TITLE
fix(ui): Use LogsViewer for container logs

### LINKED ISSUE #2569: Make log viewer compatible with output in color
# Summary

Container log viewer is not compatible with containers that use terminal escape code to display text in color.

# Proposal

Argo CD has this feature and is using the LogsViewer component from  argo-ui package.

### PR DESCRIPTION
**Pull request by dtaniwaki**
Checklist:

* [X] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this is a chore.

Fixes #2569

* [x] The title of the PR is (a) [conventional](https://www.conventionalcommits.org/en/v1.0.0/), (b) states what changed, and (c) suffixes the related issues number. E.g. `"fix(controller): Updates such and such. Fixes #1234"`.  
* [x] I've signed the CLA.
* [x] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [x] My builds are green. Try syncing with master if they are not. 
* [x] My organization is added to USERS.md.

Screen Shot 2020-04-24 at 5 19 47 PM

### CODE REVIEW

File: ui/src/app/workflows/components/workflow-logs-viewer/full-height-logs-viewer.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+
+import {LogsViewer} from 'argo-ui';
+import {LogsViewerProps} from 'argo-ui/src/components/logs-viewer/logs-viewer';
+
```
Comment (alexec) [inline ui/src/app/workflows/components/workflow-logs-viewer/full-height-logs-viewer.tsx:5]: I’d just in-line this file m

**Comment by dtaniwaki** [comment]
Screen Shot 2020-04-25 at 2 11 47 PM

I added an example file which emits colored logs in the `examples` directory.

**Comment by alexec** [comment]
Run `make docs` to fix codegen job.

**Comment by tsadoklf** [comment]
Will it be possible to configure "themes"? Different configurations for different users?

**Comment by alexec** [comment]
Can you please sync with master and then run run `make docs` to fix CI?

**Comment by alexec** [comment]
🚀 thank you!

**Pull request merged**